feature/experimental/isis/ate_tests/isis_interface_passive_test/isis_interface_passive_test.go

```diff
@@ -0,0 +1,399 @@
+			}
+			// Checking multitopology.
+			if got := gnmi.Get(t, dut, adjPath.MultiTopology().State()); got != false {
+				t.Errorf("FAIL- Expected value for multi topology not found,got %t,want %t", got, false)
+			}
+			// Checking neighbor circuit type.
+			if got := gnmi.Get(t, dut, adjPath.NeighborCircuitType().State()); got != oc.Isis_LevelType_LEVEL_2 {
+				t.Errorf("FAIL- Expected value for circuit type not found,got %s,want %s", got, oc.Isis_LevelType_LEVEL_2)
+			}
+			// Checking neighbor ipv4 address.
+			if got := gnmi.Get(t, dut, adjPath.NeighborIpv4Address().State()); got != atePort1attr.IPv4 {
+				t.Errorf("FAIL- Expected value for ipv4 address not found,got %s,want %s", got, atePort1attr.IPv4)
+			}
+			// Checking isis neighbor extended circuit id.
+			if got := gnmi.Get(t, dut, adjPath.NeighborExtendedCircuitId().State()); got == 0 {
+				t.Errorf("FAIL- Expected neighbor extended circuit id not found,expected non-zero value,got %d", got)
+			}
+			// Checking neighbor snpa.
+			snpa_address := gnmi.Get(t, dut, adjPath.NeighborSnpa().State())
+			mac, err := net.ParseMAC(snpa_address)
+			if !(mac != nil && err == nil) {
+				t.Errorf("FAIL- Expected value for snpa address not found,got %s", snpa_address)
+			}
+			// Checking isis adjacency address families.
+			if got := gnmi.Get(t, dut, adjPath.Nlpid().State()); !cmp.Equal(got, []oc.E_Adjacency_Nlpid{oc.Adjacency_Nlpid_IPV4, oc.Adjacency_Nlpid_IPV6}) {
+				t.Errorf("FAIL- Expected address families not found,got %s,want %s", got, []oc.E_Adjacency_Nlpid{oc.Adjacency_Nlpid_IPV4, oc.Adjacency_Nlpid_IPV6})
+			}
+			// Checking neighbor ipv6 address.
+			ipv6_address := gnmi.Get(t, dut, adjPath.NeighborIpv6Address().State())
+			ip := net.ParseIP(ipv6_address)
+			if !(ip != nil && ip.To16() != nil) {
+				t.Errorf("FAIL- Expected ipv6 address not found,got %s", ipv6_address)
+			}
+			// Checking isis priority.
+			_, present := gnmi.Lookup(t, dut, adjPath.Priority().State()).Val()
+			if !present {
+				t.Errorf("FAIL- Priority is not present")
+			}
+			// Checking isis restart status.
+			_, chk_present := gnmi.Lookup(t, dut, adjPath.RestartStatus().State()).Val()
+			if !chk_present {
+				t.Errorf("FAIL- Restart status not present")
+			}
+			// Checking isis restart support.
+			_, check_present := gnmi.Lookup(t, dut, adjPath.RestartSupport().State()).Val()
+			if !check_present {
+				t.Errorf("FAIL- Restart support not present")
+			}
+			// Checking isis restart suppress.
+			_, checks_present := gnmi.Lookup(t, dut, adjPath.RestartStatus().State()).Val()
+			if !checks_present {
+				t.Errorf("FAIL- Restart suppress not present")
+			}
+		})
+		t.Run("System level counter checks", func(t *testing.T) {
+			// Checking authFail counters.
+			if got := gnmi.Get(t, dut, statePath.Level(2).SystemLevelCounters().AuthFails().State()); got != 0 {
+				t.Errorf("FAIL- Not expecting any authentication key failure,got %d,want %d", got, 0)
```

Comment:
add a space after the commas in all these error messages

Reply:
taken care in latest commit